urdf/urdfdom_compatibility.h.in

```diff
@@ -69,16 +69,25 @@ URDF_TYPEDEF_CLASS_POINTER(Sphere);
 URDF_TYPEDEF_CLASS_POINTER(Visual);
 
 URDF_TYPEDEF_CLASS_POINTER(ModelInterface);
+
+typedef boost::shared_ptr<void> VoidSharedPtr;
```

Comment:
Why do you define `VoidSharedPtr `here? This is a compatibility header to declare types of `urdfdom`. However, `VoidSharedPtr `is only used internally by `ColladaParser`. Hence, the declaration should go to `collada_parser/src/collada_parser.cpp:179`.